src/main/permissionsManager.ts

```diff
@@ -152,16 +152,16 @@ export class PermissionsManager extends JsonFileManager<Permissions> {
                 detail: localizeMessage(`main.permissionsManager.checkPermission.dialog.detail.${permission}`, 'Would you like to grant {appName} this permission?', {appName: app.name}),
                 type: 'question',
                 buttons: [
-                    localizeMessage('label.allow', 'Allow'),
                     localizeMessage('label.deny', 'Deny'),
                     localizeMessage('label.denyPermanently', 'Deny Permanently'),
+                    localizeMessage('label.allow', 'Allow'),
                 ],
             });
 
             // Save their response
             const newPermission = {
-                allowed: response === 0,
-                alwaysDeny: (response === 2) ? true : undefined,
```

Comment:
Should we define these are const for readiability and also if we need to change something it will propagate to other places where we check like below?

Reply:
This is really the only place that does it, and it's based on the ordering of the options. So it's not repeatable really. I think it's okay as-is for now.

Reply:
FWIW - Below in line 174 it's also used and in tests